src/hooks/useMarginRemStyle.ts

```diff
@@ -4,6 +4,11 @@ import type { ViewStyle } from 'react-native'
 import { useTheme } from '../components/services/ThemeContext'
 
 export interface MarginRemProps {
+  /** Sets the alignment of the element within its container */
```

Comment:
Don't reinvent the wheel here, 'baseline' isn't used nor obvious what it even means. just do `FlexAlignType`